api/third_party/openapi/odpf/metadata/optimus/Job.swagger.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
This file should not be part of changes. It is deleted on this branch.
Currently swagger schema is under protos.

Reply:
done